src/game/client/prediction/entities/pickup.cpp

```diff
@@ -29,7 +29,9 @@ void CPickup::Tick()
 			switch(m_Type)
 			{
 			case POWERUP_HEALTH:
-				//pChr->Freeze();
+				if(!GameWorld()->m_WorldConfig.m_IsDDRace || !GameWorld()->m_WorldConfig.m_PredictDDRace)
```

Comment:
```suggestion
				if(!GameWorld()->m_WorldConfig.m_PredictDDRace)
```
This should only depend on whether we're predicting DDRace or not.